diff --git a/src/requests/utils.py b/src/requests/utils.py
index 6f66281..d92c4b8 100644
--- a/src/requests/utils.py
+++ b/src/requests/utils.py
@@ -483,15 +483,6 @@ def unquote_header_value(value: str, is_filename: bool = False) -> str:
     if value and value[0] == value[-1] == '"':
-        # this is not the real unquoting, but fixing this so that the
-        # RFC is met will result in bugs with internet explorer and
-        # probably some other browsers as well.  IE for example is
-        # uploading files with "C:\foo\bar.txt" as filename
         value = value[1:-1]
-
-        # if this is a filename and the starting characters look like
-        # a UNC path, then just return the value without quotes.  Using the
-        # replace sequence below on a UNC path has the effect of turning
-        # the leading double slash into a single slash and then
-        # _fix_ie_filename() doesn't work correctly.  See #458.
-        if not is_filename or value[:2] != "\\\\":
-            return value.replace("\\\\", "\\").replace('\\"', '"')
+        return value.replace("\\\\", "\\").replace('\\"', '"')
     return value
