diff --git a/src/requests/utils.py b/src/requests/utils.py
index 6f66281..f1b8d3c 100644
--- a/src/requests/utils.py
+++ b/src/requests/utils.py
@@ -393,9 +393,6 @@ def to_key_val_list(value):
     if value is None:
         return None

-    if isinstance(value, (str, bytes, bool, int)):
-        raise ValueError("cannot encode objects that are not 2-tuples")
-
     if isinstance(value, Mapping):
         value = value.items()

     return list(value)
