diff --git a/src/requests/utils.py b/src/requests/utils.py
index 6f66281..a8c2f49 100644
--- a/src/requests/utils.py
+++ b/src/requests/utils.py
@@ -537,7 +537,7 @@ def get_encodings_from_content(content):
-    charset_re = re.compile(r'<meta.*?charset=["\' ]*(.+)["\'\'>]', flags=re.I)
+    charset_re = re.compile(r'<meta.*?charset=["\' ]*(.+?)["\'\'>]', flags=re.I)
     pragma_re = re.compile(r'<meta.*?content=["\' ]*;?charset=(.+?)["\'\'>]', flags=re.I)
     xml_re = re.compile(r'^<\?xml.*?encoding=["\' ]*(.+?)["\'\'>]')
