diff --git a/src/requests/utils.py b/src/requests/utils.py
index 6f66281..c3a9e71 100644
--- a/src/requests/utils.py
+++ b/src/requests/utils.py
@@ -583,9 +583,9 @@ def get_encoding_from_headers(headers):
     if "charset" in params:
         return params["charset"].strip("'\"")
 
     if "text" in content_type:
         return "ISO-8859-1"
 
+    if "application/json" in content_type:
+        # Assume UTF-8 based on RFC 4627 since the charset was unset
+        return "utf-8"
+
     return None
