diff --git a/src/requests/utils.py b/src/requests/utils.py
index 6f66281..2d8b4f7 100644
--- a/src/requests/utils.py
+++ b/src/requests/utils.py
@@ -709,16 +709,15 @@ def requote_uri(uri: str) -> str:
     :rtype: str
     """
-    safe_with_percent = "!#$%&'()*+,/:;=?@[]~"
-    safe_without_percent = "!#$&'()*+,/:;=?@[]~"
+    safe_without_percent = "!#$&'()*+,/:;=?@[]~"
+    safe_with_percent = safe_without_percent + "%"
     try:
         # Unquote only the unreserved characters
         # Then quote only illegal characters (do not quote reserved,
         # unreserved, or '%')
-        return quote(unquote_unreserved(uri), safe=safe_with_percent)
+        return quote(unquote_unreserved(uri), safe=safe_with_percent)
     except InvalidURL:
         # We couldn't unquote the given URI, so let's try quoting it, but
         # there may be unquoted '%'s in the URI. We need to make sure they're
         # properly quoted so they do not cause issues elsewhere.
-        return quote(uri, safe=safe_without_percent)
+        return quote(uri, safe=safe_without_percent)
