diff --git a/src/flask/sessions.py b/src/flask/sessions.py
index 36e4a82..b71c9d4 100644
--- a/src/flask/sessions.py
+++ b/src/flask/sessions.py
@@ -242,9 +242,9 @@ class SessionInterface:
         .. versionadded:: 0.11
         """
 
-        return session.modified or (
-            session.permanent and app.config["SESSION_REFRESH_EACH_REQUEST"]
-        )
+        needs_refresh = session.permanent and app.config["SESSION_REFRESH_EACH_REQUEST"]
+        return session.modified or needs_refresh
